### DISCUSSION TITLE
PeerTube

### DISCUSSION THREAD

**Discussion by dr-33-m** [coollabsio/coolify#3860] [Service Template Requests]
Video streaming a.ternative to Youtube
https://github.com/chocobozzz/peertube

**Comment by ayo-run** [comment]
Yes please! Peertube is a self-hosted alternative to youtube which many creators in the Fediverse use.

**Comment by bst1n** [comment]
This !

**Comment by bst1n** [comment]
I was able to set up Peertube by following this tutorial: https://lmika.org/2025/03/01/running-peertube-in-coolify.html

I even managed to add SMTP and S3 Storage.

**Comment by TriHydera** [comment]
Bump
